docs/docs/icicle/rust-bindings/polynomials.md

```diff
@@ -151,6 +161,7 @@ use icicle_bn254::polynomials::DensePolynomial as PolynomialBn254;
 ```
 
 ### Creation
+
```

Comment:
Please remove the comment on line 168 (starting with 'Assume F is ...'). I forgot it and it doesn't make any sense since there is no F :) 
thanks